tests/test_bot.py

```diff
@@ -385,6 +400,97 @@ def test(_, url, data, *args, **kwargs):
                                                switch_pm_text='switch pm',
                                                switch_pm_parameter='start_pm')
 
+    def test_answer_inline_query_current_offset_error(self, bot, inline_results):
+        with pytest.raises(ValueError, match=('`current_offset` and `next_offset`')):
+            bot.answer_inline_query(1234,
+                                    results=inline_results,
+                                    next_offset=42,
+                                    current_offset=51)
+
+    def test_answer_inline_query_current_offset_1(self, monkeypatch, bot, inline_results):
```

Comment:
Again a case for parametrization. There are mutliple assertions in this test.

Reply:
here i see it